src/features/community/CommunityGardenEntry.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from "react";
+
+import { GRID_WIDTH_PX } from "features/game/lib/constants";
+
+import merchant from "assets/npcs/merchant.png";
+import humanSign from "assets/buildings/human_sign.png";
+import arrowUp from "assets/icons/arrow_up.png";
+import { CommunityGardenModal } from "./CommunityGardenModal";
+import { merchantAudio } from "lib/utils/sfx";
+
+export const CommunityGardenEntry: React.FC = () => {
+  const [showModal, setShowModal] = useState(false);
+
+  const openMerchant = () => {
+    merchantAudio.play();
+    setShowModal(true);
+    //Checks if merchantAudio is playing, if false, plays the sound
+    if (!merchantAudio.playing()) {
```

Comment:
There are cases when first merchantAudio.play() wont work?

Reply:
Tested it and its actually line 15 that should be removed. IsPlaying check is here to prevent overlaping sound when you open and close modal too fast.

Reply:
removed extra `merchantAudio.play()` thanks!